resources/views/modals/add-note.blade.php

```diff
@@ -0,0 +1,29 @@
+{{-- See snipeit_modals.js for what powers this --}}
+<div class="modal-dialog">
+    <div class="modal-content">
+        <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+            <h2 class="modal-title">{{ trans('general.add_note')  }}</h2>
+        </div>
+
+        <div class="modal-body">
+            <form action="{{ route('api.notes.store') }}" onsubmit="return false">
+                <input type="hidden" name="type" value="{{request("type")}}"/>
+                <input type="hidden" name="id" value="{{request("id")}}"/>
+                <div class="alert alert-danger" id="modal_error_msg" style="display:none"></div>
+
+                <div class="row">
+                    {{ Form::label('note', trans('general.add_note'), array('class' => 'col-md-3 control-label')) }}
+                    <div class="col-md-8">
+                        <textarea class="form-control" id="note" name="note">{{ old('note') }}</textarea>
+                        {!! $errors->first('note', '<span class="alert-msg" aria-hidden="true"><i class="fas fa-times" aria-hidden="true"></i> :message</span>') !!}
+                    </div>
+                </div>
+            </form>
+        </div>
+        <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">{{ trans('button.cancel') }}</button>
```

Comment:
Can we make the cancel button left-aligned, to make it consistent with the other forms and modals we use?

Reply:
done